core/test-loader.ts

```diff
@@ -2,9 +2,21 @@ import { FileRequirer, TestFixture } from "./";
 import { ITest, ITestCase, ITestFixture } from "./_interfaces";
 import { METADATA_KEYS } from "./alsatian-core";
 
+/**
+ * Class responsible to load a test file with fixtures and make TestFixtures instances.
+ */
 export class TestLoader {
+  /**
+   * Creates a new TestLoader class
+   * @param _fileRequirer A helper FileRequirer instance
+   */
   public constructor(private _fileRequirer: FileRequirer) {}
 
+  /**
+   * Loads a test file that contains one or more fixtures
+   * @param filePath The file path of the file to load the Fixture from
+   * @returns An array of the loaded fixtures
+   */
   public loadTestFixture(filePath: string): Array<ITestFixture> {
```

Comment:
This method, I think, that should be called something like `loadFile` since it can load one or more fixture. The actual method name makes think that it loads only **one** TestFixture.